impersonation/README.md

```
@@ -344,4 +344,95 @@ Returns a single impersonation request identified by its `id`.
+
+    ```json
+    { "statusCode": 404, "error": "Not Found", "message": "Request does not exist." }
+    ```
+
+  - **Code:** 401
+
+    ```json
+    { "statusCode": 401, "error": "Unauthorized", "message": "You are not authorized for this action." }
+    ```
+
+#### Authentication and Authorization
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)